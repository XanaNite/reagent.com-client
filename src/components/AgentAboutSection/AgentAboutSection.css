@media screen and (min-width: 490px) and (max-width: 768px){
    .about-container{
        display: flex;
        justify-content: space-between;
        flex-wrap: wrap;
    }

    .aboutDetails-container{
        display: flex;
        flex-wrap: wrap;
    }

    .aboutDetails-container p{
        width: 50%;
    }
}

@media screen and (min-width: 770px){
    .about-container{
        display: flex;
        justify-content: space-between;
        flex-wrap: wrap;
    }
    
    #agentBio{
        width: 45%;
    }

    .aboutDetails-container{
        width: 50%;
    }
}