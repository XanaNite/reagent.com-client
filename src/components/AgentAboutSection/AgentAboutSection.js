import React from 'react'
import './AgentAboutSection.css'
//import {multipleInputValues} from '../Utils/Utils'

export default function AgentAboutSection(props){
    return (
        <div className="agentInfo-about">
            <h3>About {props.agent.first_name} {props.agent.last_name} with {props.agent.office}</h3>
            <div className="about-container">
                <p id="agentBio">{props.agent.bio}</p>
                <div className="aboutDetails-container">
                    <p>Experience: {props.agent.experience} years</p>
                    {/*<p>Specializations: {multipleInputValues(props.agent.agentSpecialization)}</p>
                    <p>Areas Served: {multipleInputValues(props.agent.agentAreasServed)}</p>*/}
                    <p>Brokerage: {props.agent.brokerage}</p>
                    <p>Slogan: {props.agent.slogan}</p>
                    {/*<p>Designations & Certifications</p>
                    <p>Price Range (last 24 months)</p>
                    <p>Recommendations count</p>*/}
                </div>
            </div>
        </div>
    )
}