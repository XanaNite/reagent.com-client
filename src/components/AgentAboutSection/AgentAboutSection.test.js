import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AgentAboutSection from './AgentAboutSection'

describe(`AgentAboutSection component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders AgentAboutSection by default', () => {
    const wrapper = shallow(<AgentAboutSection />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders AgentAboutSection given props', () => {
    const wrapper = shallow(<AgentAboutSection {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})