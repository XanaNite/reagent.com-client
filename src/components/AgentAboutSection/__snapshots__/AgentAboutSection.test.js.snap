// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentAboutSection component renders AgentAboutSection given props 1`] = `
<div
  className="agentInfo-about"
>
  <h3>
    About 
    test
     
    test
     with 
  </h3>
  <div
    className="about-container"
  >
    <p
      id="agentBio"
    >
      Not entered
    </p>
    <div
      className="aboutDetails-container"
    >
      <p>
        Experience: 
        Not entered
         years
      </p>
      <p>
        Brokerage: 
        Not entered
      </p>
      <p>
        Slogan: 
        Not entered
      </p>
    </div>
  </div>
  <div
    className="agentInfo-contact"
  >
    <h4>
      Contact Me
    </h4>
    <p>
      Phone: 
       
    </p>
    <p>
      Email: 
    </p>
  </div>
</div>
`;
